tests/e2e/specs/modules/adsense/setup-new-user.test.js

```diff
@@ -254,4 +260,71 @@ describe( 'setting up the AdSense module', () => {
 
 		await expect( '/' ).not.toHaveAdSenseTag();
 	} );
+
+	describe( 'AMP is setup', () => {
+		beforeEach( async () => {
+			await activateAMPWithMode( 'standard' );
+		} );
+		afterEach( async () => {
+			await deactivatePlugin( 'amp' );
+		} );
+		it( 'has valid AMP for logged-in users', async () => {
+			datapointHandlers.accounts = ( request ) => {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( [
+						ADSENSE_ACCOUNT,
+					] ),
+				} );
+			};
+
+			datapointHandlers.clients = ( request ) => {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( [
+						{
+							arcOptIn: false,
+							id: `ca-${ ADSENSE_ACCOUNT.id }`,
+							kind: 'adsense#adClient',
+							productCode: 'AFC',
+							supportsReporting: true,
+						},
+					] ),
+				} );
+			};
```

Comment:
It would be nice to extract most of this to something outside the test as these response handlers are generally necessary to set up AdSense, and are not really significant to the test. Again, more of a nice-to-have, not a blocker.

Reply:
@aaemnnosttv I've left this as-is for now and will open a new issue to create module setup helpers for e2e tests.